src/components/wrapper/index.js

```diff
@@ -57,22 +55,20 @@ export class Wrapper extends Component {
             className={classnames(
               'pipeline-sidebar__hide-menu pipeline-icon-button',
               {
-                'pipeline-sidebar__hide-menu--offset': !showHistory,
                 'pipeline-sidebar__hide-menu--visible': visibleNav
               }
             )}
             onClick={this.toggleNav.bind(this)}>
             <Icon type="close" title="Close" theme={theme} />
           </button>
-          <SidebarTabs />
+          <ChartUI />
```

Comment:
Could the naming of this be better? i.e. ChartUI isn't very semantically named (don't really know what i'm getting when I use it etc)

Reply:
What would you recommend? `<SidebarContent />`?

Reply:
I think something like SidebarFilters would be a good name for it. I'm about to completely redesign this section, so I'll address the naming in a subsequent PR.